workflows/proteomics/clinicalmp/clinicalmp-verification/iwc_clinicalmp_verification_workflow_tests.yml

```diff
@@ -0,0 +1,49 @@
+- doc: Test outline for iwc_clinicalmp_verification_workflow
+  job:
+    SGPS_peptide-report:
+      class: File
+      path: test-data/SGPS_peptide-report.tabular
+      filetype: tabular
+    Distinct_Peptides_for_PepQuery:
+      class: File
+      path: test-data/Distinct_Peptides_for_PepQuery.tabular
+      filetype: tabular
+    MaxQuant-peptide-report:
+      class: File
+      path: test-data/MaxQuant-peptide-report.tabular
+      filetype: tabular
+    Input_MGFs_Dataset_Collection:
```

Comment:
there's no need to include `Dataset_Collection` in the name here.

Reply:
Changed to Input MGFs